src/platform/testing/unit/mocha-setup.js

```diff
@@ -63,7 +63,18 @@ export default function setupJSDom() {
 
   win.dataLayer = [];
   win.scrollTo = () => {};
-  win.sessionStorage = {};
+  Object.defineProperty(win, 'sessionStorage', {
```

Comment:
The `sessionStorage` and `localStorage` props that JSDOM creates are now readonly which creates issues for some of our tests that try to replace them. This override fixes that.